impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -118,8 +120,12 @@ struct TexImage2DData {
         external_format = GL_RGBA;
         type = GL_HALF_FLOAT;
         break;
-      case PixelFormat::kUnknown:
       case PixelFormat::kS8UInt:
+        internal_format = GL_DEPTH_STENCIL;
```

Comment:
I think this is a combined depth stencil format, like kD24UnormS8Uint below. This is fine to use now but we should investigate if there is a stencil-only format.

Reply:
Sg, I'll add it the MSAA next steps issue and added a TODO here.